trunner/target/emulated.py

```diff
@@ -80,9 +85,22 @@ def from_context(cls, _: TestContext):
         return cls()
 
 
+class SPARCV8Leon3GenericQemuTarget(QemuTarget):
+    name = "sparcv8leon3-generic-qemu"
+    rootfs = False
+
```

Comment:
```suggestion
    rootfs = False
    experimental = True
```

And then include `sparcv8leon3-generic-qemu` to psh tests